src/charm.py

```diff
@@ -126,9 +139,6 @@ def _render_config(self, config):
                 "/etc/nginx/sites-enabled/{}".format(site_conf_name),
             )
 
-    def _reload_config(self):
```

Comment:
I moved this to a separate function because it's a self contained function that doesn't need an instance of the class.  This does make the diff larger though; please let me know if I should do a more minimal change first, then open a PR with some refactoring later.

Reply:
I think it's good enough to just change this function because:

1. > This charm is under maintenance mode. Only critical bug will be handled.
2. we should use https://github.com/canonical/operator-libs-linux/blob/main/lib/charms/operator_libs_linux/v1/systemd.py#L210

Reply:
Ah my bad, I didn't know it's only under maintenance mode. :/  Sure I'll just change the function.

Reply:
Updated :)